src/flight_control/av_state.h

```diff
@@ -22,7 +22,9 @@ enum class State
     ERRORFLIGHT,
     THRUSTSEQUENCE,
     MANUAL,
-    ARMED
+    ARMED,
+    READY,
+    LIFTOFF
```

Comment:
Might be better to put them in chronological nominal order. (THRUST_SEQUENCE between READY and LIFTOFF).